pkg/api/server/v1alpha2/result/result_test.go

```diff
@@ -212,3 +214,21 @@ func TestFormatName(t *testing.T) {
 		t.Errorf("want %s, got %s", want, got)
 	}
 }
+
+func TestGetResultByParentName(t *testing.T) {
```

Comment:
This is a case where we probably don't need this test, since the GetResult test fully covers the logic contained here, and is a simple enough wrapper around a (soon to be) exported func that we probably doesn't make sense to break out into its own test.